plugins/polish/novelki.ts

```diff
@@ -9,18 +9,28 @@ class NovelkiPL implements Plugin.PluginBase {
     name = "Novelki";
     icon = "src/pl/novelki/icon.png";
     site = "https://novelki.pl";
-    version = "1.0.2";
-    filters: Filters | undefined = undefined;
+    version = "1.0.3";
 
     async popularNovels(
         page: number,
-        { showLatestNovels }: Plugin.PopularNovelsOptions,
+        { filters }: Plugin.PopularNovelsOptions<typeof this.filters>
     ): Promise<Plugin.NovelItem[]> {
-        const body = await fetchApi(`${this.site}/projekty?page=${page}`).then((res) => res.text());
-        const loadedCheerio = parseHTML(body);
+        let link = this.site + "/projekty?filter=t"
+        link += '&genre=' + filters.genres.value;
+        link += '&status=' + filters.status.value;
+        link += '&type=' + filters.type.value;
```

Comment:
if filter value is `""`, it is still valid? 
the url will be like `https://site.com/novels?filter=t&genre=&status=&type=`